### PR TITLE
Search 161 3.7

### PR DESCRIPTION
**Pull request by alexbakharew**
### Scope & Purpose

*(Please describe the changes in this PR for reviewers - **mandatory**)*

- [ ] :hankey: Bugfix (requires CHANGELOG entry)
- [ ] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [ ] :book: CHANGELOG entry made

#### Backports:

- [ ] No backports required
- [ ] Backports required for: *(Please specify versions and link PRs)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket number:
- [ ] Design document: 

### Testing & Verification

*(Please pick either of the following options)*

- [ ] This change is a trivial rework / code cleanup without any test coverage.
- [ ] The behavior in this PR was *manually tested*
- [ ] This change is already covered by existing tests, such as *(please describe tests)*.
- [ ] This PR adds tests that were used to verify all changes:
  - [ ] Added new C++ **Unit tests**
  - [ ] Added new **integration tests** (e.g. in shell_server / shell_server_aql)
  - [ ] Added new **resilience tests** (only if the feature is impacted by failovers)
- [ ] There are tests in an external testing repository:
- [ ] I ensured this code runs with ASan / TSan or other static verification tools

Link to Jenkins PR run:

### Documentation

> All new features should be accompanied by corresponding documentation. 
> Bugs and features should furthermore be documented in the CHANGELOG so that
> support, end users, and other developers have a concise overview. 

- [ ] Added entry to *Release Notes* 
- [ ] Added a new section in the *Manual* 
- [ ] Added a new section in the *HTTP API* 
- [ ] Added *Swagger examples* for the HTTP API  
- [ ] Updated license information in *LICENSES-OTHER-COMPONENTS.md* for 3rd party libraries

### External contributors / CLA Note 

Please note that for legal reasons we require you to sign the [Contributor Agreement](https://www.arangodb.com/documents/cla.pdf)
before we can accept your pull requests.

### CODE REVIEW

**Review by gnusi — APPROVED** [review]

File: CHANGELOG [hunk 1]
```diff
@@ -2,6 +2,10 @@ v3.7.13 (XXXX-XX-XX)
 --------------------
 
 * Updated arangosync to 1.6.0.
+* Add prefix parameter to LEVENSHTEIN_MATCH function in ArangoSearch
+
+* Removed redirects from /_admin/cluster* to /_admin/cluster/*. Adjusted
```
Comment (gnusi) [inline CHANGELOG:7]: completely unrelated to this PR

File: tests/Aql/DecaysFunctionTest.cpp [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+#include <velocypack/velocypack-aliases.h>
+
+using namespace arangodb;
+using namespace arangodb::aql;
+using namespace arangodb::containers;
+
+namespace {
+
+// helper functions
+SmallVector<AqlValue> createArgVec(const VPackSlice slice) {
```
Comment (gnusi) [inline tests/Aql/DecaysFunctionTest.cpp:51]: completely unrelated to this PR

File: tests/IResearch/IResearchFeature-test.cpp [hunk 3]
```diff
@@ -1576,7 +1576,11 @@ TEST_F(IResearchFeatureTest, test_start) {
     { "MIN_MATCH", { ".,.|.+", FunctionType::FILTER } },
     { "LIKE", { ".,.|.", FunctionType::FILTER } },
     { "NGRAM_MATCH", { ".,.|.,.", FunctionType::FILTER } },
-    { "LEVENSHTEIN_MATCH", { ".,.,.|.,.", FunctionType::FILTER } },
+    { "LEVENSHTEIN_MATCH", { ".,.,.|.,.,.", FunctionType::FILTER } },
+    { "IN_RANGE", { ".,.,.,.,.", FunctionType::FILTER } },
+    { "GEO_IN_RANGE", { ".,.,.,.|.,.,.", FunctionType::FILTER } },
```
Comment (gnusi) [inline tests/IResearch/IResearchFeature-test.cpp:1581]: completely unrelated to this PR

**Review by gnusi — CHANGES_REQUESTED** [review]

**Review by gnusi — APPROVED** [review]

**Review by KVS85 — COMMENTED** [review]
LGTM.

**Pull request merged**